src/components/helper/Offset.vue

```diff
@@ -9,39 +9,43 @@
 </template>
 
 <script>
-export default {
-  props: {
-    fullWidth: {
-      type: Boolean,
-      default: false
-    },
-    offset: {
-      type: [Number, String],
-      default: 0
-    }
-  },
+/* eslint-disable */
+  export default {
+    name: 'HelperOffset',
 
-  computed: {
-    classes () {
-      return {
-        'v-offset--full-width': this.fullWidth
+    props: {
+      fullWidth: {
+        type: Boolean,
+        default: false
+      },
+      offset: {
+        type: [Number, String],
+        default: 0
       }
     },
-    styles () {
-      return {
-        top: `-${this.offset}px`,
-        marginBottom: `-${this.offset}px`
+
+    computed: {
+      classes () {
+        return {
+          'v-offset--full-width': this.fullWidth
+        }
+      },
+      styles () {
+        return {
+          top: `-${this.offset}px`,
+          marginBottom: `-${this.offset}px`
+        }
       }
     }
   }
-}
 </script>
 
 <style>
 .v-offset {
   margin: 0 auto;
   max-width: calc(100% - 32px);
   position: relative;
+  z-index: 1;
```

Comment:
Any particular reason for adding this? And for the choice of a low value?

Reply:
Not sure. This file is from the theme (creative tom vuetify dashboard). So I just copied it as-is (I should have commented which files needed more attention in the review - sorry)